### DISCUSSION TITLE
Need help with launching fastify website locally

### DISCUSSION THREAD

**Discussion by msbn21** [fastify/fastify#4949] [General]
I am following the instructions provided in fastify/website repo to launch the website locally on my machine, however `npm start` command keeps complaining about missing `static/generated/benchmarks.json` file while running `docusaurus start`, can someone please help me with this ?

**Comment by Eomm** [comment]
Please, let's move the discussion here https://github.com/fastify/website/issues/46

Anyway, you need to run the build first: https://github.com/fastify/website#build
